caproto/server/stats.py

```diff
@@ -76,20 +82,24 @@ def __init__(self, *args, **kwargs):
 
     engineer = pvproperty(
         value='',
+        max_length=40,
         name='ENGINEER',
         record='stringin',
         read_only=True,
+        doc='Who is responsible this abomination',
```

Comment:
```suggestion
        doc='Who is responsible for this abomination',
```
I only review jokes.

Reply:
Just under 40 chars; I'll accept it